<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KPayeesViewDecl</class>
 <widget class="QWidget" name="KPayeesViewDecl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="m_splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="KListWidget" name="m_payeesList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
       <widget class="KTabWidget" name="m_tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Transactions</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QGroupBox" name="transactionGroup">
            <property name="title">
             <string>Transactions</string>
            </property>
            <layout class="QVBoxLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>11</number>
             </property>
             <item>
              <layout class="QVBoxLayout">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="K3ListView" name="m_transactionView">
                 <property name="allColumnsShowFocus">
                  <bool>true</bool>
                 </property>
                 <property name="shadeSortColumn">
                  <bool>false</bool>
                 </property>
                 <column>
                  <property name="text">
                   <string>No.</string>
                  </property>
                  <property name="clickable">
                   <bool>true</bool>
                  </property>
                  <property name="resizable">
                   <bool>true</bool>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Date</string>
                  </property>
                  <property name="clickable">
                   <bool>true</bool>
                  </property>
                  <property name="resizable">
                   <bool>true</bool>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Category</string>
                  </property>
                  <property name="clickable">
                   <bool>true</bool>
                  </property>
                  <property name="resizable">
                   <bool>true</bool>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Amount</string>
                  </property>
                  <property name="clickable">
                   <bool>true</bool>
                  </property>
                  <property name="resizable">
                   <bool>true</bool>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="m_balanceLabel">
                 <property name="text">
                  <string>Balance: </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab1">
         <attribute name="title">
          <string comment="Street address">Address</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QGroupBox" name="detailsGroup">
            <property name="title">
             <string>Payee Information</string>
            </property>
            <layout class="QGridLayout">
             <property name="margin">
              <number>11</number>
             </property>
             <property name="spacing">
              <number>6</number>
             </property>
             <item row="3" column="0">
              <widget class="QLabel" name="TextLabel7">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string comment="E-mail address">E-Mail:</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="TextLabel5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Postal Code:</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="TextLabel6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Telephone/Fax:</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="TextLabel4_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Notes</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="KLineEdit" name="postcodeEdit"/>
             </item>
             <item row="2" column="1">
              <widget class="KLineEdit" name="telephoneEdit"/>
             </item>
             <item row="3" column="1">
              <widget class="KLineEdit" name="emailEdit"/>
             </item>
             <item row="0" column="1">
              <widget class="KTextEdit" name="addressEdit" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>1</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="KTextEdit" name="notesEdit" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>1</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="TextLabel4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string comment="Street address">Address:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="spacer6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="TabPage">
         <attribute name="title">
          <string>Matching</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="m_matchTypeGroupBox">
            <property name="title">
             <string>Transaction Matching</string>
            </property>
            <layout class="QVBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="radioNoMatch">
               <property name="text">
                <string>No matching</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string>m_matchType</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioNameMatch">
               <property name="text">
                <string>Match on Payee name</string>
               </property>
               <attribute name="buttonGroup">
                <string>m_matchType</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioKeyMatch">
               <property name="text">
                <string>Match on a name listed below</string>
               </property>
               <attribute name="buttonGroup">
                <string>m_matchType</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="KEditListBox" name="matchKeyEditList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis">
             <string>This list contains the names that will match this payee if a transaction is imported from an external source. Keep in mind, that you can specify regular expressions here.</string>
            </property>
            <property name="title">
             <string>List of matching names</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkMatchIgnoreCase">
            <property name="text">
             <string>Ignore Case</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="TabPage2">
         <attribute name="title">
          <string>Default Account</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QCheckBox" name="checkEnableDefaultAccount">
            <property name="text">
             <string>Use the default account for
new transactions with this payee</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="labelDefaultAccount">
              <property name="text">
               <string>Default category:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="KMyMoneyAccountCombo" name="comboDefaultAccount"/>
            </item>
            <item row="1" column="1">
             <widget class="KPushButton" name="buttonSelectMyAccount">
              <property name="text">
               <string>Suggest a category</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="spacer5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>31</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="KPushButton" name="m_helpButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer14">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>260</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="KPushButton" name="m_updateButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyAccountCombo</class>
   <extends>KComboBox</extends>
   <header>kmymoneyaccountcombo.h</header>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>K3ListView</class>
   <extends>Q3ListView</extends>
   <header>k3listview.h</header>
  </customwidget>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">../widgets/kmymoneyaccountcombo.h</include>
 </includes>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="m_matchType"/>
 </buttongroups>
</ui>
